specs/AppNotifications/AppNotifications-spec.md

```diff
@@ -543,6 +543,367 @@ namespace Microsoft.Windows.AppNotifications
 }
 ```
 
+## Enhanced UX Notification for Video and Audio Call
+The Enhanced UI for Calling Notifications introduces a series of design improvements aimed at making video call notifications clearer, more interactive, and user-friendly. This feature is designed to increase the likelihood of users responding to calls promptly and efficiently by optimizing how notifications are presented and interacted with. We are adding improvements below to video calling UI/UX.  
+ 
+#### Interactive Elements: 
+
+- Quick Actions: Integrated buttons for mic and camera to allow users to manage their audio device and camera directly from the notification without needing to switch apps or windows. 
+ 
+- Live camera feed preview: Users can view a live feed of themselves when they make/receive a video call. This will help users be better prepared for video calls and also potentially avoid embarrassing situations.
+
+A screenshot of a Audio and video call
+
+![Notification Progress](Enhanced_ux_video_call_Notifications.png)
+
+Screenshot 1: Displays an example of Audio Call with new element as the setting button on click of that buttons it should show devices list for Audio Input and Output Devices. 
+
+Screenshot 2 and 3: Displays an example of Audio Call with 2 new elements - Camera Preview and Setting Button. Setting button click shows devices list for Audio Input and Output Devices. 
+
+If the camera device is changed, then the camera preview element should also display the selected device preview. 
```

Comment:
Also, mention that camera preview would be displayed locally on the user machine and camera feed data on notification toast is not sent to app

Reply:
Permissions: It will check for both app and notifications camera access permissions